src/pages/_app.tsx

```diff
@@ -17,7 +21,78 @@ const MyApp: FunctionComponent<AppProps> = ({
     GOVUKFrontend.initAll();
   }, []);
 
-  return <Component {...pageProps} />;
+  return (
+    <>
+      <Head>
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1, viewport-fit=cover"
+        />
+        <meta charSet="utf-8" />
```

Comment:
You seem to have missed off the `title` tag from here: https://design-system.service.gov.uk/styles/page-template/ is that on purpose?

Reply:
As discussed, title tag set on each page using the NextJS [Head](https://nextjs.org/docs/api-reference/next/head) component.